minecraft_servers/vultex/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+  "server_name": "vultex",
+  "nice_name": "Vultex",
+  "direct_ip": "vultex.gg",
+  "server_wildcards": [
+    "%.vultexpvp.com",
+    "%.vultex.gg"
+  ],
+  "social": {
+    "discord": "discord.vultexpvp.com"

```

Comment:
Please update to "https://discord.vultexpvp.com" ([as in the manifest.json example](https://github.com/LabyMod/server-media/blob/master/README.md#example-manifestjson))

Reply:
Done.